web/core/components/global/product-updates/jsxConverter.tsx

```
@@ -0,0 +1,274 @@
+    const container = containerRef.current;
+
+    const walker = document.createTreeWalker(
+      container,
+      NodeFilter.SHOW_TEXT, // Only process text nodes
+      null // No custom filter function needed
+      // false // deprecated argument
+    );
+
+    let node;
+    while ((node = walker.nextNode())) {
+      if (node.nodeValue && node.nodeValue.includes("\u00A0")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)